src/main/java/org/gatorgradle/task/GatorGradleReportTask.java

```diff
@@ -0,0 +1,123 @@
+package org.gatorgradle.task;
+
+import org.gatorgradle.task.GatorGradleTask;
+import org.gatorgradle.command.BasicCommand;
+import org.gatorgradle.command.Command;
+import org.gatorgradle.config.GatorGradleConfig;
+import org.gatorgradle.display.CommandOutputSummary;
+import org.gatorgradle.internal.DependencyManager;
+import org.gatorgradle.internal.ProgressLoggerWrapper;
+import org.gatorgradle.util.Console;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.workers.WorkerExecutor;
+
+public class GatorGradleReportTask extends GatorGradleTask {
+
+  @Inject
+  public GatorGradleReportTask(WorkerExecutor executor) {
+    this.executor = executor;
+  }
+
+  /**
+   * Execute the grading checks assigned to this GatorGradleReportTask.
+   */
+  @TaskAction
+  public void grade() {
+
+    config.parseHeader();
```

Comment:
You should create a new method (or better yet, an `act` method in the super class) that will do the dependency management; then call that in both subclasses (use `super.act()` to reference the super class' method). We should also probably have a discussion on whether the startup command should be executed for the report task -- I don't really know if it should or shouldn't.